modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+Transparent huge pages active                      true          true             Warning   true
+```
+
+[[redpanda-lmr]]
+=== Redpanda's latest version
+
+Check that Redpanda is running on the xref:https://github.com/redpanda-data/redpanda/releases[latest maintenance release^] on every node.
+
+Example:
+```bash
+/usr/bin/redpanda --version
+```
+
+Output:
+```bash
+v23.3.9 - f0a24fa4b60d55bd075c291e213807b5dfa74c0a
+```
+
+[[cpu-memory]]
+=== Correct CPUs and Memory configured
+
+Ensure that the correct number of CPUs and memory are being used by Redpanda.
+
+Example:
+
+```bash
+journalctl -u redpanda | grep "System resources"
+```
+
+Output:
+
+```bash
+Mar 25 12:16:18 ip-172-31-10-199 rpk[3957]: INFO  2024-03-25 12:16:18,105 [shard 0:main] main - application.cc:350 - System resources: { cpus: 8, available memory: 55.578GiB, reserved memory: 3.890GiB}
+```
+
+[[mounted-disks]]
+=== Disks are correctly mounted
+
+Ensure that the correct disks are mounted and if multiple devices are used, they are configured as RAID-0. Other RAID configurations can have significantly worse latencies. File system should be of type XFS. If XFS is unavailable, ext4 is an appropriate alternative. 
+
+Example:
+```bash
+grep data_directory /etc/redpanda/redpanda.yaml
+    data_directory: /var/lib/redpanda/data
+df -khT /var/lib/redpanda/data
+```
+
+Output:
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/nvme0n1   xfs   1.8T   14G  1.8T   1% /mnt/vectorized
+```
+
+Output:
+
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/md0       xfs    14T   99G   14T   1% /mnt/vectorized
+```
+
+Output:
+```bash
+mdadm: An option must be given to set the mode before a second device (/dev/md0) is listed
+```
+
+Example:
+```bash
+mdadm --detail /dev/md0
+```
+
+Output:
+```bash
+/dev/md0:
+           Version : 1.2
+     Creation Time : Thu Apr 18 11:03:46 2024
+        Raid Level : raid0
+        Array Size : 14648172544 (13969.59 GiB 14999.73 GB)
+      Raid Devices : 2
+     Total Devices : 2
+       Persistence : Superblock is persistent
+
+       Update Time : Thu Apr 18 11:03:46 2024
+             State : clean
+    Active Devices : 2
+   Working Devices : 2
+    Failed Devices : 0
+     Spare Devices : 0
+
+            Layout : -unknown-
+        Chunk Size : 512K
+
+Consistency Policy : none
+
+              Name : ip-172-31-24-82:0  (local to host ip-172-31-24-82)
+              UUID : e9574118:10d562bf:ed3ca2d9:68ccc3a6
+            Events : 0
+
+    Number   Major   Minor   RaidDevice State
+       0     259        2        0      active sync   /dev/nvme2n1
+       1     259        0        1      active sync   /dev/nvme1n1
+```
+
+[[auth-enable]]
+=== Authentication enabled
+
+Ensure that authentication is set up (or other mitigations are in place). Without SASL authentication enabled, anybody can potentially connect to the Redpanda brokers.
```

Comment:
`authentication` is missing a hyperlink to https://docs.redpanda.com/current/manage/security/authentication/

Reply:
We're still missing a lot of links to existing pages. Still working on these

Reply:
"(or other mitigations are in place)" is vague...unless we are going to list them, I recommend removing this caveat because is is unclear/unhelpful. So either expand to explain or remove.

Reply:
@hcoyote @tmgstevens agree with @Feediver1 here. Do we have links to these mitigations? If not, let's not be vague and remove this passage.